src/components/App/App.tsx

```diff
@@ -0,0 +1,230 @@
+            title: <>Arts&Crafts festivals</>,
+            description: (
+              <>
+                Art festivals, farmers <br /> and craft markets
+              </>
+            ),
+          },
+          {
+            title: <>And many more!</>,
+            description: (
+              <>
+                Outdoor movies, live music, <br /> DJ services, and more!
+              </>
+            ),
+          },
+        ];
+
+  const staffingItems =
+    windowWidth > 800
+      ? [
+          {
+            title: (
+              <>
+                1.
+                <br /> Planning the event, <br /> managing everything <br />
+                from logistics to decor
+              </>
+            ),
+            description: (
+              <>
+                to ensure a seamless <br /> and memorable experience.
+              </>
+            ),
+          },
+          {
+            title: (
+              <>
+                2.
+                <br /> Post-event support <br /> to ensure everything <br />{" "}
+                wraps up smoothly
+              </>
+            ),
+            description: (
+              <>
+                Our commitment doesn't end <br /> when the event does.
+              </>
+            ),
+          },
+          {
+            title: (
+              <>
+                3.
+                <br />
+                We tailor every event <br /> to your theme <br /> or request
+              </>
+            ),
+            description: (
+              <>
+                We pride ourselves on our individual <br /> approach to event
+                organizing
+              </>
+            ),
+          },
+        ]
+      : [
+          {
+            title: (
+              <>
+                1.
+                <br /> Planning the event, <br /> managing everything <br />
+                from logistics to decor
+              </>
+            ),
+            description: (
+              <>
+                to ensure a seamless <br /> and memorable experience.
+              </>
+            ),
+          },
+          {
+            title: (
+              <>
+                2.
+                <br /> Post-event support <br /> to ensure everything <br />{" "}
+                wraps up smoothly
+              </>
+            ),
+            description: (
+              <>
+                Our commitment doesn't end <br /> when the event does.
+              </>
+            ),
+          },
+          {
+            title: (
+              <>
+                3.
+                <br />
+                We tailor every event <br /> to your theme or <br /> request
+              </>
+            ),
+            description: (
+              <>
+                We pride ourselves on our individual <br /> approach to event
+                organizing
+              </>
+            ),
+          },
+        ];
+
+  return (
+    <div>
+      {/* Home Page List */}
```

Comment:
Hey Joshua so I've observed that the layout breaks at 1130px, with text overflowing from the container. This issue might stem from our Tailwind CSS breakpoints. We should should probably update the breakpoints in tailwind.config.js and revise the styles in the HorizontalList component to prevent overflow. After making these tweaks and testing the sizes I'm sure we can merge the changes to main as soon as possibe.
<img width="1440" alt="Screenshot 2024-09-18 at 10 50 59 PM" src="https://github.com/user-attachments/assets/a4557d56-2b7a-4abf-b880-58091ec1b237">